### PR TITLE
.json() returns a string of used parameters

### PR DESCRIPTION
**Pull request by ShiboSoftwareDev**
image

### CODE REVIEW

**Comment by ShiboSoftwareDev** [comment]
TODO:
- return this:
image
instead of this:
image
- maybe some default values should be default inside of the component definition like this one:
image

**Comment by ShiboSoftwareDev** [comment]
image
I haven't updated this type definition, should I?

**Comment by ShiboSoftwareDev** [comment]
I have updated the format and it's still failing

File: src/fn/bga.ts [hunk 1]
```diff
@@ -134,5 +137,5 @@ export const bga = (params: BgaDefInput): AnySoupElement[] => {
     }
   }
 
-  return [...pads]
+  return { circuitJson: [...pads], parameters: JSON.stringify(params) }
```
Comment (seveibar) [inline src/fn/bga.ts:140]: Why stringify?

```suggestion
  return { circuitJson: [...pads], parameters: params }
```

**Review by seveibar — COMMENTED** [review]
Looks good except how parameters are stringified

**Comment by ShiboSoftwareDev** [comment]
@seveibar 
image

**Review by seveibar — APPROVED** [review]

**Pull request merged**